instance/cephmetrics_example.conf

```diff
@@ -0,0 +1,54 @@
+# The name of the project, user and role to create and use
+PROJECT_NAME=cephmetrics
+PROJECT_USER=cephmetrics
+PROJECT_ROLE=user
+# The password to be used for the user created above
+PROJECT_USER_PASSWORD=changeme
+# The name of the instance to create
+SERVER_NAME=cephmetrics
+# The name of the image to use (This must be present in the overcloud!)
+IMAGE_NAME=rhel_7
+# The username to use for the SSH connection to the instance
+USER_NAME=cloud-user
+# The name of the keypair to use; the private key will be created as:
+#   $HOME/$KEYPAIR_NAME.pem
+KEYPAIR_NAME=cephmetrics
+# The name of the flavor to use when creating the instance
+FLAVOR_NAME=m1.medium
+# TODO: Drop the other FLAVOR_* options, and assume the flavor exists
```

Comment:
I don't think the flavors are created by default anymore. You could make the flavor --private so it only exists within the ceph-metrics project and normal users wouldn't see it. You could also mention in the doc that users could increase available resources for the metrics instance by adjusting the flavor properties in the template. I guess my point is, you may want to keep the flavor so it works by default. 

Either way, it's a TODO and nothing to block merging this.